Apps/W1/ServiceDeclaration/app/src/Processing/ServiceDeclarationMgt.Codeunit.al

```diff
@@ -322,7 +322,7 @@ codeunit 5012 "Service Declaration Mgt."
         if not IsFeatureEnabled() then
             exit;
         if Item.Type = Item.Type::Service then begin
```

Comment:
Why are we not just exiting here, if the SalesHeader is temporary? Would that not do the trick? Calling GetSalesHeader comes at a cost (does more than getting and invokes events).

Reply:
See my comment below.

This PR does fix issues with Standard Sales Lines, too.

Additional we maybe can improve this in the future if we get a new method to simple calculate the Sales or Purch Prices for a Customer) Vendor.
Then we maybe won't need the TemSalesHeader anymore.